metadata-editor/app/controllers/EditsApi.scala

```diff
@@ -42,7 +43,8 @@ object EditsApi extends Controller with ArgoHelpers {
     // their images can be imported by drag and drop instead
     // FIXME: Creating new instances? Rubbish ಠ_ಠ.
     val usageRightsData =
-      List(PrImage(), Handout(), Screengrab(), SocialMedia(), Obituary(), Pool())
+      List(PrImage(), Handout(), Screengrab(), SocialMedia(), Obituary(), Pool(),
+           StaffPhotographer("?", "?"))
```

Comment:
I've added a note explaining why I do that - I'll keep searching for a better way as I'm not massively happy with the above examples.